### DISCUSSION TITLE
Not Accepting OAuth 2.0 Credential File (Authentication Failure) [Solved]

### DISCUSSION THREAD

**Discussion by mayanksh3khar** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#723] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/52963184/170716338-22ee5c19-7f17-4d44-8fdb-6c5f3de839a7.png)

Just set it up. I have already tried a million things but it still isn't accepting my credentials file.
I have created multiple credentials but it still isn't working with any of them. Using v1 release

**Comment by LucasCordoba** [comment]
I have the same problem

**Reply by mayanksh3khar** [reply]
Solved. Check my answer. Lemme know if it works.

**Comment by APHEX-BOTO** [comment]
same here.

**Reply by mayanksh3khar** [reply]
Solved. Check my answer. Lemme know if it works.

**Comment by andreiGl** [comment]
Yeap, same. Stuck in the middle of the setup process :(

**Reply by mayanksh3khar** [reply]
Solved. Check my answer. Lemme know if it works.

**Comment by mayanksh3khar** [comment, marked as answer]
I had to use the secret key that Cloud Console generated instead of the one I created.
Instructions: https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/671#issuecomment-1065211934
And then I had to manually copy token from web browser as the page isn't generated
Instructions: https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/671#issuecomment-1072699291
